FeatureFiles/Amazon.feature

```diff
@@ -0,0 +1,9 @@
+﻿Feature: Amazon
+	
+Scenario:Navigating to the application
+	
+	Given Opening the Chrome browser
+	When I enter the Url 
+	And I Searhc books in the searchbox
```

Comment:
All spell mistakes are treated as critical in BDD syntax based languages like Gherkin